src/registry/HttpClientRegistry.ts

```diff
@@ -134,7 +134,7 @@ export class HttpClientRegistry implements IRegistry {
     if (filter?.label) {
       queryParams.set('label', filter.label);
     }
-    return this.fetchJson<WarpRouteConfigMap>(`/warp-routes/core?${queryParams.toString()}`);
+    return this.fetchJson<WarpRouteConfigMap>(`/warp-route/core?${queryParams.toString()}`);
```

Comment:
in the repo it's under `deployments/warp_routes` tho

is this an expected divergence between the repo and the api?

Reply:
ah yes i see other places use `/warp-route` https://github.com/hyperlane-xyz/hyperlane-registry/blob/5cc34e1e7d2c456e770d36f95d7caa6eeccc507e/src/registry/HttpClientRegistry.ts#L126